presto-main/src/main/java/com/facebook/presto/operator/scalar/TryCastFunction.java

```diff
@@ -53,7 +53,7 @@ public TryCastFunction()
     @Override
     public boolean isHidden()
     {
-        return true;
+        return false;
```

Comment:
This function was hidden because it's not a regular function, but a language feature. Internally, the planner implements its by generating invocations to this magic function, but the existence of the function is just that -- an implementation detail.